web/tls_config.go

```diff
@@ -214,6 +214,11 @@ func ListenAndServe(server *http.Server, flags *FlagConfig, logger log.Logger) e
 		}
 		return ServeMultiple(listeners, server, flags, logger)
 	}
+
+	if flags.WebListenAddresses == nil {
+		*flags.WebListenAddresses = []string{":9090"}
+	}
```

Comment:
We definitely should not do this, as some users do not want any web listeners, only systemd socket.

Reply:
I agree, in addition to it being highly presumptuous for a library (as opposed to an app) to choose a default listener address.

Reply:
+1

Reply:
I definitely agree, I've implemented an error log instead. Cheers! :)